frontend/src/components/summary/changeRequest/MTFISummary.vue

```diff
@@ -273,229 +245,201 @@
-                      <v-text-field
-                        v-model.number="item.approvedFeeFeb"
-                        type="number"
-                        density="compact"
-                        flat
-                        variant="solo"
-                        hide-details
-                        readonly
-                        label="February"
-                        prefix="$"
-                      />
-                    </v-col>
-                    <v-col class="col-4"> Feb 20{{ item.programYear?.slice(-5, -3) }} </v-col>
-                  </v-row>
-
-                  <v-row no-gutters class="feeTitle summary-label justify-start ma-0">
-                    <v-col class="col-8">
-                      <v-text-field
-                        v-model.number="item.approvedFeeMar"
-                        type="number"
-                        density="compact"
-                        flat
-                        variant="solo"
-                        hide-details
-                        readonly
-                        label="March"
-                        prefix="$"
-                      />
-                    </v-col>
-                    <v-col class="col-4"> Mar 20{{ item.programYear?.slice(-5, -3) }} </v-col>
-                  </v-row>
-
-                  <!-- End Row One of Grid-->
+                  <div no-gutters class="feeTitle summary-label justify-start ma-0">
+                    Oct
+                    <v-text-field
+                      v-model.number="item.approvedFeeOct"
+                      type="number"
+                      density="compact"
+                      flat
+                      variant="solo"
+                      hide-details
+                      readonly
+                      prefix="$"
+                    />
+                  </div>
+
+                  <div no-gutters class="feeTitle summary-label justify-start ma-0">
+                    Nov
+                    <v-text-field
+                      v-model.number="item.approvedFeeNov"
+                      type="number"
+                      density="compact"
+                      flat
+                      variant="solo"
+                      hide-details
+                      readonly
+                      prefix="$"
+                    />
+                  </div>
+
+                  <div no-gutters class="feeTitle summary-label justify-start ma-0">
+                    Dec
+                    <v-text-field
+                      v-model.number="item.approvedFeeDec"
+                      type="number"
+                      density="compact"
+                      flat
+                      variant="solo"
+                      hide-details
+                      readonly
+                      prefix="$"
+                    />
+                  </div>
+
+                  <div no-gutters class="feeTitle summary-label justify-start ma-0">
+                    Jen
```

Comment:
I love to have a Jen's month but maybe we should use "Jan"? :D

Reply:
Probably ahahah